crypto/fipsmodule/aes/asm/aesni-x86.pl

```diff
@@ -153,17 +153,19 @@ sub aesni_generate1	# fully unrolled loop
 	&cmp		($rounds,11);
 	&jb		(&label("${p}128"));
 	&lea		($key,&DWP(0x20,$key));
-	&je		(&label("${p}192"));
-	&lea		($key,&DWP(0x20,$key));
+	# 192-bit key support was removed
+
+	# 14 rounds
 	eval"&aes${p}	($inout,$rndkey1)";
 	&$movekey	($rndkey1,&QWP(-0x40,$key));
 	eval"&aes${p}	($inout,$rndkey0)";
 	&$movekey	($rndkey0,&QWP(-0x30,$key));
-    &set_label("${p}192");
```

Comment:
My (intended) strategy with my own commits like this was to add a "192-bit key support was removed." comment at every point where I removed the code, which would mean that there should be one here. These comments help me do the merges from upstream when upstream changes the code.